src/library_webgl.js

```diff
@@ -739,6 +766,65 @@ for (/**@suppress{duplicate}*/var i = 0; i < {{{ GL_POOL_TEMP_BUFFERS_SIZE }}};
       disableHalfFloatExtensionIfBroken(ctx);
 #endif
 
+      // If end user enables *glGetProcAddress() functionality, then we must filter out
+      // all future WebGL extensions from being passed to the user, and only restrict to advertising
+      // extensions that the *glGetProcAddress() function knows to handle.
+#if GL_ENABLE_GET_PROC_ADDRESS
+      ctx._allSupportedExtensions = ctx.getSupportedExtensions;
+      const supportedExtensionsForGetProcAddress = [
```

Comment:
We are still generally avoiding `let` and `const` in system libraries, for closure/code size reasons.

Reply:
check, changed to `var`